docs/_layouts/default.html

```diff
@@ -58,5 +59,13 @@
     </div>
     <script src="//cdn.jsdelivr.net/jquery/2.1.1/jquery.min.js"></script>
     <script src="//cdn.jsdelivr.net/bootstrap/3.2.0/js/bootstrap.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/anchor-js/3.2.2/anchor.min.js"></script>
```

Comment:
Other links comes form jsdeliver why you choose cloudflare? https://cdn.jsdelivr.net/anchorjs/3.2.2/anchor.min.js